canonical-web-and-design/charmed-kubeflow.io
# Contact us form fixes

## Done

- Fix interactive form

## QA

- Check https://charmed-kubeflow-io-98.demos.haus/ scroll down to the bottom of the page to find the "Contact us" button
- Click on it and fill up the form, press next.
- Make sure the complete flow works

## Issue / Card

Fixes https://github.com/canonical-web-and-design/charmed-kubeflow.io/issues/97
